### ISSUE TITLE
Docker build shield show "manual" instead of "automated"?

### ISSUE THREAD

**Issue by rfgamaral** [badges/shields#2533]
Are you experiencing an issue with...
 - [x] [shields.io](https://shields.io/#/)

:beetle: **Description**
I have this docker image being built automatically by Docker Hub Automated Builds but the badge displays "manual" instead of "automated".

:link: **Link to the badge**
https://github.com/rfgamaral/docker-alpine/tree/rfgamaral/alpine-base

**Comment by PyvesB** [comment]
Hello @rfgamaral,

When querying the Docker Hub API, I get the following response:
````
{
	"user": "rfgamaral",
	"name": "alpine-base",
	"namespace": "rfgamaral",
	"repository_type": "image",
	"status": 1,
	"description": "An Alpine Linux base image, featuring s6 process management to run just about anything! ",
	"is_private": false,
	"is_automated": false,
	"can_edit": false,
	"star_count": 0,
	"pull_count": 13,
	"last_updated": "2018-12-14T18:29:28.033625Z",
	"is_migrated": false,
	"has_starred": false,
	"full_description": "...",
	"affiliation": null,
	"permissions": {
		"read": true,
		"write": false,
		"admin": false
	}
}
````

We are using the value of `is_automated` to derive what the badge is showing, as far as I can tell Shields is doing the right thing here. Have you set up the build automation properly? Might there be a consistency delay with the Docker API? Maybe you need to retrigger a build? Unfortunately, I'm not an expert about how Docker Hub works, I'm not going to be able to provide much more assistance.

Cheers,

Pyves

**Comment by rfgamaral** [comment]
> Unfortunately, I'm not an expert about how Docker Hub works, I'm not going to be able to provide much more assistance.

Nor am I, just started playing with this today...

> Have you set up the build automation properly?

As far as I can tell, yes... Every time I push a new tag a build is automatically triggered, it's working as expected. Why `is_automated` returns `false`, I don't know either.

**Comment by rfgamaral** [comment]
Well, it might be a bug on Docker side because of this: https://blog.docker.com/2018/12/the-new-docker-hub/

> New Automated Builds
> * Speed up builds using Build Caching
> * Add environment variables and run tests in your builds
> * Add automated builds to existing repositories

It's new stuff and maybe they are not updating the `is_automated` flag properly on this new build system.

I have no idea how can I contact Docker about reporting this issue, does anyone know the proper place/contact for things like this?

**Comment by PyvesB** [comment]
[This repository](https://github.com/docker/hub-feedback) seems to be the place where to report issues about the Docker Hub website itself. However, I think the API lives in [this repository](https://github.com/moby/moby), so it may also be useful to have a dig around there as well.

**Comment by rfgamaral** [comment]
Thanks, will look into it.

**Comment by jsmedmar** [comment]
Same issue here. Also getting `repo not found` for https://img.shields.io/docker/build/papaemmelab/toil_mutect.svg. Repo does [exists][https://hub.docker.com/r/papaemmelab/toil_mutect].

**Comment by jsmedmar** [comment]
i created this ticket at docker hub, feel free to complement https://github.com/docker/hub-feedback/issues/1733.

**Comment by Nevensky** [comment]
same here

**Comment by ldez** [comment]
Related to #1400

**Comment by jsmedmar** [comment]
This was actually solved by @ldez in #1400, fix is:

## Automated

```diff
-https://img.shields.io/docker/automated/{user}/{repo}.svg
+https://img.shields.io/docker/cloud/automated/{user}/{repo}.svg
```

## Status 
```diff
-https://img.shields.io/docker/build/{user}/{repo}.svg
+https://img.shields.io/docker/cloud/build/{user}/{repo}.svg
```
